app/src/main/java/com/example/roomies/ManageAccountFragment.java

```diff
@@ -0,0 +1,171 @@
+package com.example.roomies;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.net.Uri;
+import android.os.Bundle;
+
+import androidx.fragment.app.Fragment;
+
+import android.provider.MediaStore;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.Toast;
+
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+import com.parse.Parse;
+import com.parse.ParseException;
+import com.parse.ParseFile;
+import com.parse.ParseUser;
+import com.parse.SaveCallback;
+
+import java.io.ByteArrayOutputStream;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Use the {@link ManageAccountFragment#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class ManageAccountFragment extends Fragment {
+    private ImageView ivProfileImage;
+    private ImageView ivAddPhoto;
+    private EditText etNameInput;
+    private EditText etEmailInput;
+    private Button btnUpdate;
+    private Bitmap bitmap;
+
+    public static final int GET_FROM_GALLERY = 3;
+
+    public ManageAccountFragment() {
+        // Required empty public constructor
+    }
+
+    /**
+     * Use this factory method to create a new instance of
+     * this fragment using the provided parameters.
+     *
+     * @return A new instance of fragment ManageAccountFragment.
+     */
+    public static ManageAccountFragment newInstance() {
+        ManageAccountFragment fragment = new ManageAccountFragment();
+        Bundle args = new Bundle();
+        fragment.setArguments(args);
+        return fragment;
+    }
+
+    @Override
+    public void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        View view = inflater.inflate(R.layout.fragment_manage_account, container, false);
+
+        // show profile image
+        ivProfileImage = view.findViewById(R.id.ivProfileImage);
+        if (ParseUser.getCurrentUser().getParseFile("image") != null) {
+            Glide.with(this).load(ParseUser.getCurrentUser().getParseFile("image").getUrl()).apply(RequestOptions.circleCropTransform()).into(ivProfileImage);
```

Comment:
Can we break this down, for better readability? For instance, 

useImage = ParseUser.getCurrentUser().getParseFile("image").getUrl(); 
Glide.with(this).load(userImage)... etc.